test/puppetlabs/trapperkeeper/services/status/status_service_test.clj

```diff
@@ -151,6 +157,23 @@
                 "message" "No status information found for service notfound"}
               (json/parse-string (slurp (:body resp)))))))))
 
+(deftest status-code-test
+  (testing "returns 503 response code when a service is not running"
+    (with-status-service app
+      [bar-service fail-service]
+      (let [{:keys [status]} (http-client/get "http://localhost:8180/status/v1/services/fail")]
+        (is (= 503 status)))
+      (let [{:keys [status]} (http-client/get "http://localhost:8180/status/v1/services")]
+        (is (= 503 status)))))
```

Comment:
Could we add a test here that validates we get a 200 for `/status/v1/services/bar`?  (Assuming we agree that's the desired behavior :) )